ajax/libs/field-kit/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "field-kit",
+  "filename": "field-kit.min.js",
+  "version": "2.0.4",
+  "description": "Utilities to effectively manage text entry on the web.",
+  "homepage": "https://square.github.io/field-kit/",
+  "keywords": [
+    "field-kit",
+    "input",
+    "format"
+  ],
+  "author": "Square, Inc.",
+  "license": "Apache-2.0",
+  "npmName": "field-kit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
```

Comment:
"*.+(js|map)"
@iamJoeTaylor i think we also need `field-kit.min.js.map` and `field-kit.js.map`